util/Pos/Util/QuickCheck/Property.hs

```diff
@@ -62,15 +62,20 @@ qcIsRight :: Show a => Either a b -> Property
 qcIsRight (Right _) = property True
 qcIsRight (Left x)  = qcFail ("expected Right, got Left (" <> show x <> ")")
 
+-- TBD would it not be sufficient to just use 'Traversable'? Does the
+-- 'Container' class from 'universum' add any value?
```

Comment:
It's *kinda* like `mono-traversable`'s class in that you can use `fold` and friends on Text, ByteString, etc